manta-benchmark/wasm-bench.sh

```diff
@@ -0,0 +1,5 @@
+#RUSTFLAGS="-C target-feature=+simd128" wasm-pack build --release
```

Comment:
If this is unused, remove it, but if it's useful then make a note in the `README.md`.